pages/Christmas-Items.md

```diff
@@ -18,11 +18,11 @@ These items consist of mostly foods.
 
 
 ## Christmas Present
-The Christmas Present is even more special. Because it has the Santa's Magic!<br>
-When you open 1 of these, you can earn a random gift.
+The Christmas Present is even more special because it has Santa's Magic!<br>
+When you open 1 of these, you obtain random gift.
```

Comment:
```suggestion
When you open 1 of these, you obtain a random item.
```
Obtaining a random gift here sounds sort of weird because gift is a synonym for a present, and opening a present to get a gift is weird